www/jury/statistics.php

```diff
@@ -18,6 +15,10 @@
 	$probid = (int)$_GET['probid'];
 }
 
+if ( !empty($_GET['unfrozen']) ) {
```

Comment:
This option is not currently 'reachable' from the interface, is it?

Reply:
Fixed.